api/src/client/ApiFormationClient.ts

```diff
@@ -251,7 +265,7 @@ export const ApiFormationClient = (config: ApiConfig, logger: LogWriterType): Fo
           BusinessName: formationFormData.businessName,
           BusinessDesignator: formationFormData.businessSuffix as Exclude<BusinessSuffix, undefined>,
           Naic: naicsCode,
-          PracticesLaw: isForeignCorp ? false : undefined,
+          PracticesLaw: isForeignCorp ? "Yes" : undefined,
```

Comment:
Re my comment in the test file. Looks like we hardcoded this to "Yes" temporarily and never came back to actually set this value based on the formationFormData. This should be updated

Reply:
👀